frontend/src/component/admin/users/UsersList/ResetPassword/ResetPassword.tsx

```diff
@@ -0,0 +1,122 @@
+    showDialog,
+    closeDialog,
+    user,
+}: IChangePasswordProps) => {
+    const { classes: themeStyles } = useThemeStyles();
+    const { resetPassword } = useAdminUsersApi();
+    const { setToastApiError } = useToast();
+    const [resetLink, setResetLink] = useState('');
+
+    const submit = async (event: React.SyntheticEvent) => {
+        event.preventDefault();
+        if (!user.email) {
+            setToastApiError('User without email cannot reset password');
+            return;
+        }
+
+        try {
+            const token = await resetPassword(user.email).then((res) =>
+                res.ok ? res.json() : undefined,
+            );
+
+            if (token) {
+                setResetLink(token.resetPasswordUrl);
+            } else {
+                setToastApiError("Could not reset password");
+            }
+        } catch (error) {
+            setToastApiError(formatUnknownError(error));
+        }
+    };
+
+    const onCancel = (event: React.SyntheticEvent) => {
+        event.preventDefault();
+        closeDialog();
+    };
+
+    const closeConfirm = () => {
+        setResetLink('');
+        closeDialog();
+    };
+
+    return (
+        <Dialogue
+            open={showDialog}
+            onClick={submit}
+            style={modalStyles}
+            onClose={onCancel}
+            primaryButtonText='Generate Link'
+            title='Reset password'
+            secondaryButtonText='Cancel'
+            maxWidth='xs'
+        >
+            <form
+                onSubmit={submit}
+                className={classnames(
+                    themeStyles.contentSpacingY,
+                    themeStyles.flexColumn,
+                )}
+            >
+                <Typography variant='subtitle1'>
+                    Reseting password for user
+                </Typography>
+                <div className={themeStyles.flexRow}>
+                    <StyledUserAvatar user={user} variant='rounded' />
+                    <Typography
+                        variant='subtitle1'
+                        style={{ marginLeft: '1rem' }}
+                    >
+                        {user.username || user.email}
+                    </Typography>
+                </div>
+            </form>
+
+            <Dialogue
+                open={Boolean(resetLink)}
+                onClick={closeConfirm}
+                primaryButtonText='Close'
+                title='Reset password link created'
+            >
+                <Box>
+                    <Typography variant='body1' sx={{ mb: 2 }}>
+                        Using this link the user can reset the password.
+                    </Typography>
+                    <Typography variant='body1'>
+                        Provide them with the following link to reset their
+                        password. This will allow them to set up a new password
+                        and get back to using their Unleash account.
+                    </Typography>
```

Comment:
How about something like this? I'm not sure we need both paragraphs (they both say the same), and it might be worth adding a note that it shouldn't be shared with any third parties 💁🏼

```suggestion
                    <Typography variant='body1'>
                        The user can use this link to reset their password. 
                        You should not share this link with anyone but the 
                        user in question.
                    </Typography>
```

Reply:
I liked the suggestion